docs/extend.rst

```diff
@@ -30,6 +30,21 @@ To install the memcached extension you would do the following:
 3. `vagrant provision`.
 
 
+Globally installing extensions
+------------------------------
+
+In addition to per-project, extensions can be globally installed. This is useful
+for development tools that you want to apply to all your Chassis boxes, which
+aren't specific to any project.
+
+Chassis loads global extensions from ``~/.chassis/extensions`` in the same way
+that it loads project-specific extensions.
+
```

Comment:
I'm thinking we should probably add an example for reference. e.g. `git clone https://github.com/Chassis/Mailhog ~/.chassis/extensions/mailhog`